sql-cli/tests/test_connections.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from sql_cli.connections import _load_yaml_connections, validate_connections
+
+
+def test_validate_connections(caplog):
+    validate_connections()
+
+    postgres_conn_id = "postgres_conn"
+    postgres_conn_id_formatted_string = postgres_conn_id + " " * (25 - len(postgres_conn_id))
+    failed_connection_log = f"Validating connection {postgres_conn_id_formatted_string} FAILED"
+    assert failed_connection_log in caplog.text
+
+    sqlite_conn_id = "sqlite_conn"
+    sqlite_conn_id_formatted_string = sqlite_conn_id + " " * (25 - len(sqlite_conn_id))
+    added_connection_log = f"Validating connection {sqlite_conn_id_formatted_string} PASSED"
+    assert added_connection_log in caplog.text
+
+
+def test_specific_connection_with_env_vars_in_config(caplog):
```

Comment:
Is this an integration test? If yes, could we mock the communication with Redshift and change it to a unittest?

Reply:
Refactored the tests. Communications are mocked now.